lib/pdc/config.rb

```diff
@@ -118,7 +118,10 @@ def reset_base_connection
       faraday_config = {
         url:      api_url,
         headers:  PDC::Request.default_headers,
-        ssl:      ssl_config
+        ssl:      ssl_config,
+        request:  {
+          params_encoder: Faraday::FlatParamsEncoder
```

Comment:
None of the commit messages explain why this change is needed. Could you please amend the commit that the reason why this has been added?

Reply:
Yes, add a comment on that. This mentioned on https://github.com/lostisland/faraday#changing-how-parameters-are-serialized